### PR TITLE
style legend filter buttons

### LINKED ISSUE #2278: Button state enhancement for Rates over time card
## Is your feature request related to a problem? Please describe.
Users may be having an inaccurate understanding of the button's functionality when it is in an inactive state. In the screenshot, the data being shown is those from "All Groups." As such, the `Show all groups` button should be in an active, focused state instead of as disabled and non-clickable. Subsequently, the `Show highest/lowest averages` should be in a temporary, inactive state, but as such clearly communicates to users that the button can become active under certain conditions or user interactions.
Screenshot 2023-07-12 at 1 44 42 PM

## Describe the solution you'd like
Modify the existing button component to replace the current "disabled-state" with an active-state and the current "active-state" to an inactive-state to accurately reflect the button's functionality. This change will help avoid confusion and improve the user experience by providing clear visual cues regarding button availability and interactivity.

## Additional context / screenshots
By enhancing the button states, we aim to better align with user expectations and industry best practices. This change will contribute to a more intuitive interface and enhance the overall usability of the project.

### PR DESCRIPTION
**Pull request by kccrtv**
# Description and Motivation
fixes #2278

## Has this been tested? How?
tests passing

## Screenshots (if appropriate)
Screenshot 2024-02-25 at 10 24 19 PM
Screenshot 2024-02-25 at 10 24 14 PM
Screenshot 2024-02-25 at 10 24 56 PM

## Types of changes
- New UI

### CODE REVIEW

File: frontend/src/charts/trendsChart/FilterLegend.tsx [hunk 1]
```diff
@@ -54,42 +54,45 @@ export function FilterLegend({
...
-          }`}
-          onClick={() => {
-            handleClick(null)
-          }} // clear selected groups on click
-        >
-          Show all groups
-        </button>
+        <div className='mx-4 flex items-center justify-center rounded-2xl border-none border-altGreen shadow-raised-tighter'>
+          <button
+            // aria-disabled={groupsAreMinMax}
```
Comment (eriwarr) [inline frontend/src/charts/trendsChart/FilterLegend.tsx:64]: Did you mean to leave this as a comment?

**Review by eriwarr — CHANGES_REQUESTED** [review]
Looks great! Love the new button.

**Review by eriwarr — APPROVED** [review]
Awesome! Thanks, Kim!

File: frontend/src/styles/DesignTokens.ts [hunk 2]
```diff
@@ -53,6 +53,7 @@ const het = {
   greyDark: '#222',
   greyGridColor: '#f9f9f9',
   hexShareIconGray: '#757575',
+  hoverAltGreen: 'rgba(11, 82, 64, 0.08)',
```
Comment (benhammondmusic) [inline frontend/src/styles/DesignTokens.ts:56]: @kccrtv is this color actually used?

**Pull request merged**